presto-docs/src/main/sphinx/release/release-0.255.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.255
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where regular expression functions were not interruptible and could keep running for a long time after the query they were a part of failed.
+* Add interruption for runaway splits blocked in known situations controlled by ``task.interrupt-runaway-splits-timeout`` property which defaults to ``600s``.
+* Add support for querying non-lowercase table names in Druid connector (:pr:`15920`).
+* Added :func:`array_normalize` function to normalize an array by dividing each element by the p-norm of it.
+* Added connection param "protocols" that allows a user to specify which HTTP protocols the Presto client is allowed to use.
```

Comment:
protocols in double back quotes.

Also - move this to a separate section - `JDBC Changes` - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections